### ISSUE TITLE
When the user selects "Enter focus mode" button, on next tab key press , focus should go to the first line of the editor, but the focus is going to "Previous" button

### ISSUE THREAD

**Issue by LadyNaggaga** [microsoft/monaco-editor#1163]
.NET Online REPL  is having the following accessibility issue.  Please see this [issue](https://github.com/dotnet/try/issues/103) . I have also included the details below 

> **Environment Details:** 
> URL: https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=1
> Browser: Edge 42.17134.1.0 , Chrome 67.0.3396.87
> 
> **Repro Steps:**
> 1)Hit the URL : https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=1
> 2)Tab till "Enter focus mode" button and press enter
> 3)Editor pane will open up
> 4)Press tab key to check for next tab stop
> 
> **Actual Result:**
> When the user selects "Enter focus mode" button, on next tab key press , focus should go to the first line of the editor, but the focus is going to "Previous" button
> 
> **Expected Result:**
> When the user selects "Enter focus mode" button, on next tab key press , focus should go to the first line of the editor
> 
> **User Impact:**
> Keyboard only users may get confused
> 
> **MAS Reference:**
> [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9%7D)
> [Focus Order Issue foe Enter focus mode.pptx](https://github.com/dotnet/try/files/2188257/Focus.Order.Issue.foe.Enter.focus.mode.pptx)

**Comment by alexdima** [comment]
I cannot reproduce in the editor at https://microsoft.github.io/monaco-editor/ . The Tab key reaches the editor.
